tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -1032,43 +1030,45 @@ def test_list_recommendations_notification_codes(cluster_type: str):
             assert data['status'] == SUCCESS_STATUS
             assert data['message'] == UPDATE_RESULTS_SUCCESS_MSG
 
-            if j > 95:
-                response = update_recommendations(experiment_name, None, end_time)
-                data = response.json()
-                assert response.status_code == SUCCESS_STATUS_CODE
-                assert data[0]['experiment_name'] == experiment_name
+            response = update_recommendations(experiment_name, None, end_time)
+            data = response.json()
```

Comment:
Also add an assert for the expected message to validate only the first data point doesn't have recommendations and rest has.

Reply:
I don't see all the test scenarios mentioned in the issue #1059 covered here, please add negative tests and tests for all the terms